File: 2798_NumberOfEmployees/solution.py
class Solution(object):
    def numberOfEmployeesWhoMetTarget(self, hours, target):
        val = 0
        for i in hours:
            if (i >= target):
                val += 1
        return val
        """
        :type hours: List[int]
        :type target: int
        :rtype: int
        """
        

File: 2798_NumberOfEmployees/solution.dart
class Solution {
  int numberOfEmployeesWhoMetTarget(List<int> hours, int target) {
    int val = 0;
    for(int i = 0; i < hours.length; i++){
        if(hours[i] >= target){
            val++;
        }
    }
    return val;
  }
}
